products/identity/Spikes & Decisions/2024-08-15_interstitial-spike.md

```diff
@@ -0,0 +1,79 @@
+# Interstitial Spike
+
+## Description
+
+The purpose of this document is to record the technical analysis of what user interfaces (UI) can be served to users who need to migrate from their legacy DS Logon and My HealtheVet accounts to modern Login.gov or ID.me "verified" accounts.
+
+> [Spike ticket - VI-300](https://jira.devops.va.gov/browse/VI-300)
+
+## Technical Analysis
+
+- **Can we interrupt the flow of users who have logged in with MyHealtheVet, but no further - identifiers?**
+
+  - Unauthenticated
+    - Possible: Yes
+    - How: On button click in the sign in modal/page
+  - Authenticated
+    - Possible: Yes
+    - How: Lookup in Redux `user.profile.signIn.serviceName`
+
+- **Can we interrupt the flow of users who have logged in with DSLogon, but no further - identifiers?**
+
+  - Unauthenticated
+    - Possible: Yes
+    - How: On button click in the sign in modal/page
+  - Authenticated
+    - Possible: Yes
+    - How: Lookup in Redux `user.profile.signIn.serviceName`
+
+- **Can we interrupt the flow of users who have logged in with MyHealtheVet AND who are known to have a Login.gov/ID.me account that they are not using?**
```

Comment:
im not a big fan of this question/request. Where did it come from? Perhaps talking more about opinions on how this may or may not work from a user experience standpoint is warranted but in this document we can leave it at the technical level.

Reply:
These were questions that came from the ticket that Parker created. I think both should be warranted and documented.